### PR TITLE
Ensure /etc/foreman exists in acceptance test for Candlepin

### PR DESCRIPTION
**Pull request by ehelms**
Results from https://github.com/theforeman/puppet-certs/pull/371

### CODE REVIEW

File: spec/setup_acceptance_node.pp [hunk 1]
```diff
@@ -12,6 +12,10 @@
   ensure => present,
 }
 
+file { '/etc/foreman':
+  ensure => directory
+}
```
Comment (wbclark) [inline spec/setup_acceptance_node.pp:17]: I am not easily finding information about puppet's defaults for owner, group, and mode with posix provider.... will this be `root:root` and `755` ?

Comment (ekohl) [reply spec/setup_acceptance_node.pp:17]: I think it's derived from the user running the process then and the umask. In practice this means root:root and probably 755 but possibly 775. It is why I like being explicit in our production code: predictable and explicit.

Comment (wbclark) [reply spec/setup_acceptance_node.pp:17]: I saw that this is included in https://github.com/theforeman/puppet-katello/pull/423 so we know it works.

I'll leave it up to @ehelms if he thinks it's worth it to be explicit here as well

**Review by wbclark — APPROVED** [review]

File: spec/setup_acceptance_node.pp (hunk 1 above)
Comment (ehelms) [reply spec/setup_acceptance_node.pp:17]: Given this is just the testing layer, I'm good with this as it is.

**Pull request merged**